Directory.Packages.props

```
@@ -2,34 +2,36 @@
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
-    <MicrosoftExtensionsVersion>8.6.0</MicrosoftExtensionsVersion>
-    <AspireVersion>8.2.0</AspireVersion>
+    <AspireVersion>9.0.0-rc.1.24509.13</AspireVersion>
     <MicrosoftExtensionsAiVersion>9.0.0-preview.9.24507.7</MicrosoftExtensionsAiVersion>
   </PropertyGroup>
   <ItemGroup>
     <!-- Version together with Aspire -->
-    <PackageVersion Include="Aspire.Azure.Storage.Blobs" Version="$(AspireVersion)" />
-    <PackageVersion Include="Aspire.Hosting.AppHost" Version="$(AspireVersion)" />
-    <PackageVersion Include="Aspire.Hosting.Azure.Storage" Version="$(AspireVersion)" />
-    <PackageVersion Include="Aspire.Hosting.PostgreSQL" Version="$(AspireVersion)" />
-    <PackageVersion Include="Aspire.Hosting.Qdrant" Version="$(AspireVersion)" />
-    <PackageVersion Include="Aspire.Hosting.Redis" Version="$(AspireVersion)" />
-    <PackageVersion Include="Aspire.Hosting.Testing" Version="$(AspireVersion)" />
-    <PackageVersion Include="Aspire.Npgsql.EntityFrameworkCore.PostgreSQL" Version="$(AspireVersion)" />
-    <PackageVersion Include="Aspire.StackExchange.Redis" Version="$(AspireVersion)" />
+    <PackageVersion Include="Aspire.Azure.Storage.Blobs" Version="9.0.0-rc.1.24511.1" />
+    <PackageVersion Include="Aspire.AppHost.SDK" Version="$(AspireVersion)" />
+    <PackageVersion Include="Aspire.Hosting.AppHost" Version="9.0.0-rc.1.24511.1" />
+    <PackageVersion Include="Aspire.Hosting.Azure.AppContainers" Version="9.0.0-rc.1.24511.1" />
+    <PackageVersion Include="Aspire.Hosting.Azure.Storage" Version="9.0.0-rc.1.24511.1" />
+    <PackageVersion Include="Aspire.Hosting.PostgreSQL" Version="9.0.0-rc.1.24511.1" />
+    <PackageVersion Include="Aspire.Hosting.Python" Version="9.0.0-rc.1.24511.1" />
+    <PackageVersion Include="Aspire.Hosting.Qdrant" Version="9.0.0-rc.1.24511.1" />
+    <PackageVersion Include="Aspire.Hosting.Redis" Version="9.0.0-rc.1.24511.1" />
+    <PackageVersion Include="Aspire.Hosting.Testing" Version="9.0.0-rc.1.24511.1" />
+    <PackageVersion Include="Aspire.Npgsql.EntityFrameworkCore.PostgreSQL" Version="9.0.0-rc.1.24511.1" />
+    <PackageVersion Include="Aspire.StackExchange.Redis" Version="9.0.0-rc.1.24511.1" />
```

These versions aren't in sync. It would be nice to keep using the common AspireVersion property.

```suggestion
    <PackageVersion Include="Aspire.Azure.Storage.Blobs" Version="$(AspireVersion)" />
    <PackageVersion Include="Aspire.AppHost.SDK" Version="$(AspireVersion)" />
    <PackageVersion Include="Aspire.Hosting.AppHost" Version="$(AspireVersion) />
    <PackageVersion Include="Aspire.Hosting.Azure.AppContainers" Version="$(AspireVersion)" />
    <PackageVersion Include="Aspire.Hosting.Azure.Storage" Version="$(AspireVersion)" />
    <PackageVersion Include="Aspire.Hosting.PostgreSQL" Version="$(AspireVersion)" />
    <PackageVersion Include="Aspire.Hosting.Python" Version="$(AspireVersion)" />
    <PackageVersion Include="Aspire.Hosting.Qdrant" Version="$(AspireVersion)" />
    <PackageVersion Include="Aspire.Hosting.Redis" Version="$(AspireVersion)" />
    <PackageVersion Include="Aspire.Hosting.Testing" Version="$(AspireVersion)" />
    <PackageVersion Include="Aspire.Npgsql.EntityFrameworkCore.PostgreSQL" Version="$(AspireVersion)" />
    <PackageVersion Include="Aspire.StackExchange.Redis" Version="$(AspireVersion)" />
```